attest/verifier/build.rs

```diff
@@ -290,4 +81,170 @@ fn main() {
     if sgx.ias_mode() == IasMode::Development {
         cargo_emit::rustc_cfg!("feature=\"ias-dev\"");
     }
+
+    if should_generating_sim_files(&data_path) {
+        generate_sim_files(&data_path);
+    }
+}
+fn generate_sim_files(data_path: impl AsRef<Path>) {
```

Comment:
:( sorry I was hoping github would wee that this code was moved, diffing locally might show it better.